apps/auth-admin-web/components/Admin/form/ApiScopeUserCreateForm.tsx

```diff
@@ -226,10 +224,11 @@ const ApiScopeUserCreateForm: React.FC<Props> = (props: Props) => {
                         <input
                           id={scope.name}
                           type="checkbox"
-                          name={`apiScopeUser.userAccess[${scope.name}]`}
+                          {...register(
+                            `apiScopeUser.userAccess[${scope.name}]`,
+                          )}
                           className="api-scope-user-create-form__checkbox"
                           checked={checked}
-                          ref={register}
                           title={scope.name}
                           onChange={(e) =>
```

Comment:
`register` returns an `onChange` prop, which is overridden in a later prop. We must pass our onChange event handler as an option to the register function.

This happens "at least" 19 times in this PR (if I search in the github diff, but there could be more which are not visible here).

Reply:
I did run the codemod but unfortunately it just fixed the registers. I think i fixed all the onChange/onBlur issues. Could you have a look again?